src/components/Card/card.module.css

```diff
@@ -36,9 +36,14 @@
 
 .utils {
     display: flex;
+    justify-content: center;
+    align-items: center;
     gap: 8px;
 }
 
 .utils>* {
     cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
You can make a class for flex separately as you are using it in multiple places, will reduce the css% on github to some extent.

Reply:
Will do it.